.travis.yml

```diff
@@ -19,9 +19,14 @@ cache:
     - $HOME/.cache/pip
 
 python:
-  - "2.7"
-  - "3.5"
   - "3.6"
+  - "3.7"
+  - "3.8"
+
+matrix:
```

Comment:
If you bump lxml to >=4.3.5, travis will pass for 3.7 and 3.8 and you don't need this allow_failure

Reply:
The allowed_failures in 3.8 is to have it consistent accross Inveniosw modules. For now we allow the failures because the main cookiecutter does not support 3.8, but is good to start having an idea of what breaks and what doesn't so we can support is soon (hopefully really soon). For now 3.7 is priority 😃